libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKManager.kt

```diff
@@ -192,7 +201,15 @@ open class SalesforceSDKManager protected constructor(
      * The class of the activity used to perform the login process and create
      * the account.
      */
-    var loginActivityClass: Class<out Activity> = LoginActivity::class.java
+    val loginActivityClass: Class<out Activity>
+        get() {
```

Comment:
Both of these could drop their braces, return and `=` a single expression, for optional Kotlin style points.